.pipelines/custom-templates/post-publish-template.yml

```diff
@@ -0,0 +1,53 @@
+parameters:
+  - name: name
+    type: string
+  - name: libName
+    type: string
+  - name: path
+    type: string
+    default: 'lib'
+  - name: cdn
+    type: boolean
+    default: false
+
+steps:
+  - task: DownloadPipelineArtifact@2
+    condition: always()
+    inputs:
+      buildType: 'specific'
+      project: 'a7934fdd-dcde-4492-a406-7fad6ac00e17'
+      definition: '1758'
+      buildVersionToDownload: 'latest'
+      allowFailedBuilds: true
+      artifactName: 'drop_publish_publish_msal_${{ parameters.libName }}'
+      targetPath: '$(Pipeline.Workspace)/${{ parameters.libName }}-artifacts'
+    continueOnError: true
+  - task: CmdLine@2
+    displayName: 'Save ${{ parameters.libName }} CDN SRI Hashes'
+    condition: and(succeeded(), eq(${{ parameters.cdn }}, true))
+    inputs:
+      script: |
+        cp $(Pipeline.Workspace)/${{ parameters.libName }}-artifacts/README.md ${{ parameters.path}}/msal-${{ parameters.libName }}/README.md
+        cp $(Pipeline.Workspace)/${{ parameters.libName }}-artifacts/docs/cdn-usage.md ${{ parameters.path}}/msal-${{ parameters.libName }}/docs/cdn-usage.md
+  - task: CmdLine@2
+    displayName: 'Save package.json'
+    condition: succeeded()
+    inputs:
+      script: | 
+        cat $(Pipeline.Workspace)/${{ parameters.libName }}-artifacts/package.json
```

Comment:
Shouldn't this be package-lock?

Reply:
Oh I see you're updating package-lock below. What is this step doing?

Reply:
This is a good catch of something left over from testing, thanks!

Reply:
I added a save step for the package.json because it was easier to compare and see if the changes were properly reflected on a diff because it has the version at the top. Removed this step.